File: TransCoder_tokenized_test_set_functions/python/FIND_ROW_NUMBER_BINARY_MATRIX_MAXIMUM_NUMBER_1S.py
def findMax ( arr ) :
    row = 0
    j = N - 1
    for i in range ( 0, N ) :
        while ( arr [ i ] [ j ] == 1 and j >= 0 ) :
            row = i
            j -= 1
    print ( "Row number = ", row + 1, ", MaxCount = ", N - 1 - j )

File: results/java_codex/FIND_ROW_NUMBER_BINARY_MATRIX_MAXIMUM_NUMBER_1S.java
void findMax(int arr[][N])
{
    int row = 0, i, j;
    for (i = 0, j = N - 1; i < N; i++)
    {
        while (j >= 0 && arr[i][j] == 1)
        {
            row = i;
            j--;
        }
    }
    cout << "Row number = " << (row + 1);
    cout << ", MaxCount = " << (N - 1 - j);
}